swift/integration-tests/partial-modules/Sources/partial-modules/partial_modules.swift

```diff
@@ -0,0 +1,11 @@
+import A
+import B
```

Comment:
Currently, the extractor will fail at importing these modules and will emit a couple of unresolved AST nodes for the following `a.text`/`b.text` calls.